banzai/tests/utils.py

```diff
@@ -164,3 +164,7 @@ def __init__(self):
         self.frameid = 1234
         self.filepath = '/tmp/'
         self.filename = 'test.fits'
+
+
+class FakeImage(FakeCalImage):
```

Comment:
I'm not sure why this is necessary. Hopefully we can remove it?

Reply:
I added this FakeImage class because one of the test classes (qc_stage_integration_tests.py) imports it, and it did not exist. So I don't know how the current set of unit tests were passing, but this is what I had to do to get them to pass running locally using the command specified in the readme.

Reply:
I'll take a look at this and update it.